fastlane/Fastfile

```diff
@@ -3,13 +3,42 @@
 # The env variables are contained in an .env.default file.
 
 # Define the minimal Fastlane version
-fastlane_version "1.95.0"
+fastlane_version "2.35.1"
 
 # Use the iOS platform as default
 default_platform :ios
 
+# Default path formatters
+project_xcconfig_path = "%s/ConfigurationFiles/%s.xcconfig"
```

Comment:
In the file system it has no spaces. Do you think we should change it? I think it's more clear to have the groups matching the file system.